lib/ui/settings/SettingsHomePanel.dart

```diff
@@ -468,17 +519,26 @@ class _SettingsHomePanelState extends State<SettingsHomePanel> implements Notifi
   // NotificationsOptions
 
   Widget _buildNotifications() {
-    return _OptionsSection(
-      title: Localization().getStringEx("panel.settings.home.notifications.title", "Notifications"),
-      widgets: <Widget>[
-        ToggleRibbonButton(
+    List<Widget> contentList = new List();
+
+    List<dynamic> codes = FlexUI()['settings.notifications'] ?? [];
+    for (int index = 0; index < codes.length; index++) {
+      String code = codes[index];
+      BorderRadius borderRadius = _borderRadiusFromIndex(index, codes.length);
+      if (code == 'covid19') {
+        contentList.add(ToggleRibbonButton(
           height: null,
-          borderRadius: _borderRadiusFromIndex(0, 1),
+          borderRadius: borderRadius,
```

Comment:
@TodorBachvarov Please review this change is correct. It's related to a previous fix made by you. Awaiting for confirmation